pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -209,6 +209,7 @@ type WaitingPod interface {
 
 // Plugin is the parent type for all the scheduling framework plugins.
 type Plugin interface {
+	BuildArgs() interface{}
```

Comment:
@ahg-g any thoughts about this?

Reply:
since this is only for testing, why not add BuildArgs only to the plugins that actually have args? It is less robust, but I think a good compromise between ability to verify the arguments and littering the code base with functions that just return nil.

Reply:
In that case, we can have the (unexported) interface defined in `v1alpha1/framework.go`